.github/workflows/pr_checks.yml

```diff
@@ -0,0 +1,39 @@
+name: Wait for Vercel and run E2E tests
+
+on:
+  pull_request:
+    branches:
+      - develop # Run only on PRs to develop branch (temporary)
+
+jobs:
+  wait-for-vercel: # Used instead of deployment_status to allow base branch filtering - it's possible to get base.ref only for PRs - otherwise results would be inconsistent
+    name: Waiting for Vercel
```

Comment:
What do you think about making it a step of `e2e-tests` instead, to avoid additional job listed in the GitHub Checks?

Reply:
I made it a separate job to only have a concurrency lock (`concurrency: oss-e2e-tests-dev-env`) on actual tests - not this wait-for-vercel "step". It can be merged into the single job when the concurrency lock will be removed.